falcon/response_helpers.py

```diff
@@ -105,7 +105,7 @@ def format_content_disposition(value):
     #   * When a "filename" parameter is included as a fallback (as per above),
     #     "filename" should occur first, due to parsing problems in some
     #     existing implementations.
-    return "attachment; filename=%s; filename*=UTF-8''%s" % (
+    return disposition_type + "; filename=%s; filename*=UTF-8''%s" % (
```

Comment:
just a style thing. I would use '%s' also for `disposition_type`. also above

Reply:
Done.